db/migrate/20160314175436_create_comments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration
+  def change
+    create_table :comments do |t|
+      t.string :comment
+      t.belongs_to :post
```

Comment:
I would change this column in your posts table from `:comment` to maybe `:content` or `:text` so its a little more semantic